pkgs/development/tools/build-managers/cmake/check-improper-prefix-hook.sh

```diff
@@ -1,9 +1,9 @@
-cmakePcfileCheckPhase() {
+cmakeImproperPrefixUsageCheckPhase() {
     while IFS= read -rd $'\0' file; do
         grepout=$(grep --line-number '}//nix/store' "$file" || true)
         if [ -n "$grepout" ]; then
             {
-            echo "Broken paths found in a .pc file! $file"
+            echo "Broken paths found in a file! $file"
```

Comment:
```suggestion
            echo "Error! Broken paths found in: $file"
```

Reply:
Is the idea to generate a warning or to error completely?

Reply:
the original commit had "exit 1", and I do agree it's better to error out as warnings may be ignored by maintainers who don't look at build logs

Reply:
OK then. Let's make the message more greppable.

Reply:
```suggestion
            echo "cmakeImproperPrefixUsageCheckPhase: Broken paths found in a file! $file"
```

Reply:
Good name.

Reply:
what is the meaning of `>$2` here?

Reply:
Is this a "redirect to stderr"?

Reply:
Maybe a typo, `1&>2`

Reply:
yep typo

`>&2`